__tests__/__setup.js

```diff
@@ -4,7 +4,7 @@ import { dirname } from 'path-unified';
 import { fs } from 'style-dictionary/fs';
 import { chaiWtrSnapshot } from '../snapshot-plugin/chai-wtr-snapshot.js';
 import { fixDate } from './__helpers.js';
-import { writeZIP } from '../lib/utils/convertToDTCG.js';
+import { writeZIP } from '../dist/esm/utils/convertToDTCG.mjs';
```

Comment:
Eventually I would like to move the tests to TS as well and import source files rather than dist files, but we can save that for a separate PR